packages/twenty-ui/src/input/button/components/__stories__/Button.stories.tsx

```diff
@@ -197,6 +200,84 @@ export const PositionCatalog: CatalogStory<Story, typeof Button> = {
     fullWidth: { control: false },
     soon: { control: false },
     position: { control: false },
+    shortcut: { control: false },
+  },
+  parameters: {
```

Comment:
maybe we don't need to have two full catalog
the shortcut one could only have 1 dimension which is the shortcut